Cargo.toml

```diff
@@ -9,12 +9,12 @@ documentation = "https://docs.rs/substrate-bip39"
 repository = "https://github.com/paritytech/substrate-bip39"
 
 [dependencies]
-pbkdf2 = { version = "0.8.0", default-features = false }
-sha2 = "0.9.5"
+pbkdf2 = { version = "0.9.0", default-features = false }
+sha2 = "0.9.8"
 hmac = "0.11.0"
-schnorrkel = "0.9.1"
-zeroize = { version = "1.0.0", default-features = false }
+schnorrkel = "0.10.1"
```

Comment:
We don't use that in Substrate. So, we first should check this and make sure nothing breaks.

Reply:
I don't understand your answer.

Reply:
I meant that we don't use schnorrkel 0.10.1 in Substrate while that is already released since quite some time. Before we merge this, we should ensure that we can bump schnorrkel in Substrate as well and nothing breaks.